js/data.js

```diff
@@ -6,3 +6,10 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
```

Comment:
You are missing the portion to check if there is already a value in localStorage, and if there is, replace the initial value of `data` with the new value from localStorage